config/db

```diff
@@ -8,6 +8,8 @@ niceness=0
 rvm_remote_server_path1=maven2/org/jruby/jruby-dist
 rvm_remote_server_url1=https://repo1.maven.org
 rvm_remote_server_url2=https://rubies.travis-ci.org
+rvm_remote_server_path3=releases/download
+rvm_remote_server_url3=https://github.com/oracle/truffleruby
```

Comment:
This is unfortunately an issue, and I'm sure what's the best way to address it:
`rvm_remote_server_url3` is documented as a "user-defined" server URL in https://github.com/rvm/rvm/blob/aa3b553261cf91dcb23a04f2db08824b6ef36aa3/help/prepare.md
It's also used by TravisCI for example (see https://travis-ci.org/eregon/mspec/builds/429941797#L477), which probably means that would override the setting here.

What's a good way to fix this?

Reply:
I think ultimately the existing logic that handles binary downloads isn't extensible and needs to be rewritten. Another issue with `rvm_remote_server_url3` is that `rvm_remote_server_url2` appears to be breaking iteration outside the Travis environment, so this doesn't appear to be working as-is.

It would really be more of a punt, but we could use `rvm_remote_server_url0` for TruffleRuby and switch the iterator to start from zero instead of one after blank.